docs/release-procedure.md

```diff
@@ -0,0 +1,35 @@
+Dask-jobqueue heavily relies on dask and distributed upstream projects.  
+We may want to check their status while releasing.
+
+
+Release for dask-jobqueue, from within your fork:
+
+* Update release notes in `docs/source/changelog.rst`. Preferably within a PR,
+which informs of incoming release.
```

Comment:
The last sentence here isn't a full sentence.  I recommend merging them.

Submit a PR that updates the release notes in `docs/source/changelog.rst`.  We submit a PR to inform other developers of the pending release.